scripts/test_report_upload_script.py

```diff
@@ -0,0 +1,41 @@
+import boto3
```

Comment:
Just a thought: given this script is more or less the same across our repos I wonder if we could host it in a centralized repository and reuse it across all of our repos.

Reply:
Yea that seems like a good idea. I can look into this, first thing that comes to mind is to move it in Akamai enterprise git repos and check it out from there.